app/src/main/res/values/strings.xml

```diff
@@ -622,6 +622,17 @@
     <!-- Content description (not visible, for screen readers etc.): "Close button for library settings" -->
     <string name="content_description_close_button">Close</string>
 
+    <!-- Title to show in alert when a lot of tabs are to be opened
```

Comment:
I am assuming these strings already got OK'd

Reply:
Yes, these strings were reviewed and approved.